src/Grpc/test/InteropTests/InteropTests.cs

```diff
@@ -26,6 +26,7 @@ public InteropTests(ITestOutputHelper output)
         }
 
         [Theory]
+        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/22101")]
```

Comment:
@JunTaoLuo and @JamesNK can we please prioritize splitting up this `[Theory]` so that we can quarantine things more granularly? Is there an issue tracking that work?

Reply:
New issue: https://github.com/dotnet/aspnetcore/issues/22117